less/serialConsole.less

```diff
@@ -0,0 +1,78 @@
+/**
+  SerialConsole styles
+ */
+
+.console-pf > .terminal {
+  color: @color-pf-white;
+  text-align: left;
+  outline: medium none;
+  background-color: @color-pf-black;
+  border: 1px solid @color-pf-black;
+  padding: 10px;
+}
+
+.console-pf {
+  font-family: @font-family-monospace;
+  margin-top: 0;
+  margin-bottom: 0;
+  font-size: 10px;
+  text-align: center;
+  line-height: normal;
+}
+
+@media (min-width: @screen-tablet) {
+  .console-pf {
+    font-size: @font-size-small;
+  }
+}
+
+.console-pf > pre {
+  padding: 10px;
+  text-align: left;
+  display: block;
+  font-family: inherit;
+  font-size: inherit;
+  width: 48em;
+  height: 310px;
+  overflow-y: scroll;
+  white-space: pre-wrap;
+  margin: 0 auto;
+}
+
+.console-pf .terminal .terminal-cursor {
+  border: 1px solid @color-pf-white;
+}
+
+.console-pf .terminal:focus .terminal-cursor {
+  border: none;
+  animation: blink 1s step-end infinite;
+}
+
+@keyframes blink {
+  from {
+    color: @color-pf-black;
+    background: @color-pf-white;
+  }
+  50% {
+    color: @color-pf-white;
+    background: @color-pf-black;
+  }
+}
+
+.console-actions-pf {
+  float: right;
+  position: relative;
+  top: -2em;
```

Comment:
I would suggest removing these styles and just using 
```
display: flex;
justify-content: flex-end;
```

Reply:
Works fine (means stays visually same) if the SerialConsole is used alone.
The plan is to use it in a higher-level component wrapping serial console, VNC and probably Spice where recent version works better.